README.rst

```diff
@@ -0,0 +1,21 @@
+sdx-rabbit-monitor
+##################
+
+`sdx-rabbit-monitor` is a component of the Office for National Statistics (ONS) Survey Data Exchange (SDX) product which polls the RabbitMQ management API for monitoring purposes.
+
+Data returned is logged to sys.stderr.
+
+Configuration
+=============
+
+The main configuration options are listed below:
+
++---------------------------------+---------------+------------------------------------------------+
+| Environment Variable            | Default       | Description                                    |
++---------------------------------+---------------+------------------------------------------------+
+| RABBITMQ_HOST                   | `0.0.0.0`     | Host for the RabbitMQ service                  |
```

Comment:
I noticed this wasn't in the other repos... I'll raise issues there to add for consistency